docs/src/components/githubRelease.astro

```diff
@@ -4,8 +4,8 @@ import '../styles/custom.css';
 const fetchReleases = async () => {
   const latestVersion = [{
     id: 0,
-    name: "v0.1.5",
-    html_url: "https://github.com/platform9/vjailbreak/releases/tag/v0.1.5"
+    name: "v0.1.6",
```

Comment:
these should all be `0.1.7` as the PR is targeted to `release-0.1.7` branch

Reply:
it should get updates in the action. Let me make that change.